doc/README

```diff
@@ -14,22 +20,46 @@ optionally, you might need for special features:
  - flickcurl (for Flickr support)
  - libGL/sdl12 (to build slideshow viewer)
  - dbus-glib, gnome-keyring (kwallet/gnome keyring support)
- - osm-gps-map for geo tagging view
+ - osm-gps-map and libsoup for geo tagging view
  - lensfun (lens distortion plugin)
  - OpenEXR for HDR export
-
-other used packages (supplied in the source tree):
-
- - LibRaw nikon_curve (taken from ufraw)
- - RawSpeed
- - squish
+ - SDL, SDL-image and Mesa OpenGL for the slideshow view
+ - flickcurl for uploading direct to Flickr
+ - libsecret for storing passwords
+ - libcolord-dev libcolord-gtk-dev for colour profile support
+ - webp and openjpeg libraries for WebP and JPEG 2000 support
+ - GraphicsMagick library for TIFF EXIF and LDR image format support
 
 then, type:
 $ ./build.sh --prefix /usr --buildtype Release
 $ cd build && make install (or sudo make install)
 $ darktable
 
-and enjoy!
+optionally, to build the user manual:
+
+ - Java JDK, gnome-doc-utils, Saxon 6.5.x, FOP and ImageMagick
+
+then, type:
+$ cd build
+$ make darktable-usermanual
+$ evince doc/usermanual/darktable-usermanual.pdf
+
+optionally, to build translations of the manual pages:
+
+ - PO for anything (po4a)
+
+other used packages (supplied in the source tree):
+
+ - RawSpeed
+ - Lua 5.2 and LuaAutoc
+
+darktable has OpenCL support for cards with:
+
+ - > 1GB graphics RAM (more is better)
+ - a modern AMD or nVidia chipset
+ - proprietary drivers from the manufacturer loaded 
```

Comment:
proprietary drivers aren't exactly required
it's just that there is *still* no open opencl drivers that support all the stuff